shotover-proxy/src/message/mod.rs

```diff
@@ -347,13 +347,13 @@ impl Value {
                 ColType::Blob => Value::Bytes(Bytes::copy_from_slice(actual_bytes)),
                 ColType::Boolean => Value::Boolean(decode_boolean(actual_bytes).unwrap()),
                 ColType::Counter => Value::Integer(decode_int(actual_bytes).unwrap() as i64),
-                ColType::Decimal => Value::Float(decode_decimal(actual_bytes).unwrap().as_plain()),
+                ColType::Decimal => unimplemented!("We dont have a decimal type yet"),
```

Comment:
let's create some todos to support a Dec and Varint type -> This will blow up for any real world schema

Reply:
By todos, do you mean raise an issue?

Reply:
yup!

Reply:
done https://github.com/shotover/shotover-proxy/issues/392